spec/acceptance/basic_spec.rb

```diff
@@ -4,7 +4,9 @@
   it_behaves_like 'an idempotent resource' do
     let(:manifest) do
       <<-PUPPET
-      include pulpcore
+      class { 'pulpcore':
+        api_client_auth_cn_map => {${facts['networking']['fqdn']} => 'admin'},
```

Comment:
I wonder if we should set that in `spec/acceptance/hieradata/common.yaml` instead?